src/apps/aave/shortcuts.ts

```diff
@@ -32,22 +32,22 @@ const hook: ShortcutsHook = {
         name: 'Deposit',
         description: 'Lend your assets to earn interest',
         networkIds: [networkId],
-        // category: 'deposit',
+        category: 'deposit',
         triggerInputShape: {
           token: z.object({
             // TODO: consider requiring only tokenId and (decimal) amount
             // Right now it would mean more changes in hooks
             address: ZodAddressLowerCased,
             decimals: z.coerce.number(),
-            amount: z.string(), // in decimal string
           }),
+          amount: z.string(), // in decimal string
```

Comment:
Could we leave this within the `token`?
Anticipating the case where we want to deposit multiple tokens.
For instance to deposit into a standard pool with 2 assets.